sample-config/naemon.cfg.in

```diff
@@ -1014,3 +1014,13 @@ allow_empty_hostgroup_assignment=0
 # with a minimum of 4 workers.  This value will override the defaults
 
 #check_workers=3
+
+# CIRCULAR DEPENDENCIES (EXPERIMENTAL)
+# Allow for circular dependencies in naemon's host graph.
+# Enabaling this will cause propagation the following to stop working:
```

Comment:
"Enabaling", "propagation of"?.
But I think that we perhaps could be more clear and say something like:

Enabling this will cause the following to stop working:
* enabling/disabling and propagating notifications
* scheduling and propagating downtimes